packages/web-react/src/components/TextArea/TextArea.stories.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import TextArea from './TextArea';
+import { SpiritTextAreaProps } from '../../types';
+
+export default {
```

Comment:
I think that definition of `argTypes` is not necessary. Please look at `Alert` stories were` as ComponentMeta<typeof Alert>;` is used. Using this the Storybook can determine those types.

Reply:
Commented in the previous conversation: https://github.com/lmc-eu/spirit-design-system/pull/538#discussion_r1014324460.